projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -951,17 +960,33 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 			$my_site_title = esc_html__( 'My Sites', 'jetpack' );
 		}
 
-		$wp_admin_bar->add_menu(
-			array(
-				'parent' => 'root-default',
-				'id'     => 'blog',
-				'title'  => $my_site_title,
-				'href'   => $my_site_url,
-				'meta'   => array(
-					'class' => 'my-sites mb-trackable',
-				),
-			)
-		);
+		$class = 'my-sites mb-trackable';
+		if ( $this->use_dev_nav ) {
+			$class .= ' dev-mode';
```

Comment:
This extra class will allow us to target styles specifically for this change from the related css file on wpcom.